### PR TITLE
TMDM-11178 Remove license alerts from widget and remove 'Alerts' and 'Task' widget from CE version

### PR DESCRIPTION
**Pull request by yjli-talend**
**What is the current behavior?** (You can also link to an open issue here)
On welcome page,we have 'Alerts' widget to display the license status 'No license' or 'License will expire soon'.

**What is the new behavior?**
We remove license alerts from widget.'Alert' and 'Task' widget will display if login user have 'license' menu,'workflow task' menu and 'Data Stewardship' menu.Because CE version don't have these menu,the two widget don't display.But CE version shouldn't include 'Alert' and 'Task' widget,we remove them now.

**Please check if the PR fulfills these requirements**

- [x] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

**Comment by pbabin** [comment]
Task was to remove license alerts from the widget , not to remove the widget itself.
Right now we only have the license contributing to that widget, but we could add others such:
- IAM communication breakdown
- TDS communication breakdown
- Bonita Communication breakdown
etc...

File: org.talend.mdm.webapp.welcomeportal/src/main/java/org/talend/mdm/webapp/welcomeportal/client/widget/AlertPortlet.java [hunk 1]
```diff
@@ -49,46 +49,22 @@ public void refresh() {
...
-                    label.setText(MessagesFactory.getMessages().alerts_desc());
-                    sb.append("</span>"); //$NON-NLS-1$
-                    updateMessage(sb.toString());
-                }
-            }
-        });
+        label.setText(MessagesFactory.getMessages().no_alerts());
+        fieldSet.setVisible(false);
+        
+        // we don't any alert message now,we can use follow code when we have new alert message.
```
Comment (pbabin) [inline org.talend.mdm.webapp.welcomeportal/src/main/java/org/talend/mdm/webapp/welcomeportal/client/widget/AlertPortlet.java]: Remove commented code

File: org.talend.mdm.webapp.welcomeportal/src/main/java/org/talend/mdm/webapp/welcomeportal/client/MainFramePanel.java [hunk 2]
```diff
@@ -687,23 +687,9 @@ public void setPortletToLocations(Map<String, List<Integer>> portletToLocations)
...
-                    MainFramePanel.this.add(portlet);
-                } else {
-                    int index = default_index_ordering.indexOf(PortletConstants.ALERT_NAME);
-                    initializePortlet(default_index_ordering.get(index + 1));
-                }
-            }
-
-        });
-
+        BasePortlet portlet = new AlertPortlet(MainFramePanel.this);
```
Comment (pbabin) [inline org.talend.mdm.webapp.welcomeportal/src/main/java/org/talend/mdm/webapp/welcomeportal/client/MainFramePanel.java]: Seems alerts widget will be displayed in CE too

**Comment by pbabin** [comment]
Description of the changes aren't compliant with the specs, alerts widget must stay EE only

File: org.talend.mdm.webapp.welcomeportal/src/main/java/org/talend/mdm/webapp/welcomeportal/client/MainFramePanel.java [hunk 3]
```diff
@@ -687,23 +687,9 @@ public void setPortletToLocations(Map<String, List<Integer>> portletToLocations)
...
+        portlets.add(portlet);
+        MainFramePanel.this.add(portlet);
```
Comment (glzhou) [inline org.talend.mdm.webapp.welcomeportal/src/main/java/org/talend/mdm/webapp/welcomeportal/client/MainFramePanel.java]: Remove "Alert" and "Task" from the sorting list as they are not available for CE
https://github.com/Talend/tmdm-server-se/blob/2d8fece6d5e32259321943d4dc6b4151c4701de7/org.talend.mdm.webapp.welcomeportal/src/main/java/org/talend/mdm/webapp/welcomeportal/client/MainFramePanel.java#L219

**Comment by yjli-talend** [comment]
License constants in WelcomePortal.java have removed by commit of https://github.com/Talend/tmdm-server-se/pull/323/files

**Comment by glzhou** [comment]
I18n messages needs to be removed as well, such as:
https://github.com/Talend/tmdm-server-se/blob/84edb71f1bdad3f0efc6bfd579a4b7e7f0a2b888/org.talend.mdm.webapp.welcomeportal/src/main/resources/org/talend/mdm/webapp/welcomeportal/client/i18n/WelcomePortalMessages.properties#L8

**Pull request merged**